packages/@webex/plugin-meetings/src/multistream/sendSlotManager.ts

```diff
@@ -55,6 +57,32 @@ export default class SendSlotManager {
     return slot;
   }
 
+  /**
+   * Allow users to specify 'namedMediaGroups' to indicate which named media group its audio should be sent to.
+   * @param {MediaType} mediaType MediaType of the sendSlot to which the audio stream needs to be send to the media group
+   * @param {[]}namedMediaGroups - Allow users to specify 'namedMediaGroups'.
+   * @returns {void}
+   */
+  public setNamedMediaGroups(mediaType: MediaType, namedMediaGroups: NamedMediaGroup[]) {
```

Comment:
how can the client app say that they don't want to be sending any si audio anymore with this API?

Reply:
Call the function with the value member of the namedMediaGroup as zero

Reply:
this needs to be documented in the jsdoc

Reply:
Done